modules/sharedaddy/admin-sharing.css

```diff
@@ -297,6 +300,34 @@ body.settings_page_sharing .advanced input[type=submit] {
 	height:20px;
 }
 
+.preview-telegram-me .option-smart-on a.share-telegram-me:before{
+	top: 0;
+	content: '\f519';
+}
+.preview-telegram-me .option-smart-on a.share-telegram-me:hover{
+	border: none;
+	background: #2099D5;
+}
+.preview-telegram-me .option-smart-on a.share-telegram-me{
+	line-height: 19px;
+	border: none;
+	background: #2ca5e0;
+	color: #fff !important;
+}
+.preview-telegram-me .option-smart-on a.share-telegram-me span{
+	line-height: 19px;
+}
+.preview-telegram-me .option-smart-on{
+	height: 20px;
+}
+
+.preview-twitter .option-smart-on {
```

Comment:
It looks like you've readded the Twitter button CSS here. You'd need to remove it.